### DISCUSSION TITLE
Gibt es eine Liste mit kompatiblen Strommessern?

### DISCUSSION THREAD

**Discussion by Sorbat** [hoylabs/OpenDTU-OnBattery#1112] [Q&A]
Hallo, 

gibt es eine Liste mit kompatiblen Strommessern? Ich hab aktuell den Shelly 3EM verbaut, würde diesen aber gerne austauschen. 

Welche Geräte würden dafür in Frage kommen?

**Comment by Sorbat** [comment]
Okay, ich habe gerade das hier gefunden https://github.com/helgeerbe/OpenDTU-OnBattery/wiki/Power-Meter
das sieht ja schon mal gut aus, aber hat jemand z.b. den VM-3P75CT integriert bekommen oder ähnliche Geräte?

**Reply by spcqike** [reply]
der Victron arbeitet scheinbar sehr gut mit dem Venus OS. Also in der Victron Welt.

Kann man den ohne zusätzliche Software per http oder so abfragen?

mit VenusOS muss es wohl gehen, die Daten auszuwerten und zu verarbeiten, sie in ne Datenbank zu schreiben etc. Dann kann man sie sicher auch per MQTT weiterleiten und hier benutzen. Aber ohne Umweg über VenusOS geht es bisher wohl nicht. zumindest hab ich nichts gefunden. 

in openDTU-onBattery ist es bisher auch nicht integriert.

Warum möchtest du den Shelly wechseln, und warum gegen einen (bisher nicht kompatiblen) Victron?

**Reply by Sorbat** [reply]
Ich hatte auf Facebook gesehen, dass bei jemanden der 3EM in der Verteilung explodiert ist und hab mir dazu erklären lassen, dass das Problem beim 3EM wohl die 3 Dioden sind. Der Pro hat nur noch eine und wäre damit eine bessere Wahl.
Ich würde gerne einfach den 3EM ausbauen und durch etwas anderes ersetzen, um dieses Risiko auszuschließen.
Einen Pro hätte ich hier liegen, das wäre eine Möglichkeit, nur mehrfach die Shellys ein und aus zubauen ist halt auch nicht optimal. Einen Tibber Pulse hab ich ebenfalls hier, da wollte ich gleich mal schauen ob ich den nach der Anleitung oben konfiguriert bekomme.
Den Victron fände ich ganz cool, weil er nen Lan Anschluss hat und so schön blau ist. Und blau ist bekanntermaßen ja immer besser 😄

> Kann man den ohne zusätzliche Software per http oder so abfragen?

das weiß ich nicht, ich hab noch keinen bestellt

Programmieren kann ich allerdings nicht, Venus habe ich auch nicht. 
Wenn das mit dem victron bisher nicht geht, würde ich halt übergangsweise den Pulse oder den pro nutzen.



![image](https://github.com/user-attachments/assets/250ce777-f7ce-4b31-a05e-21b9f0da9c0c)
![image](https://github.com/user-attachments/assets/24923102-6ea1-4d11-987e-e5c3b378ce73)
![image](https://github.com/user-attachments/assets/85d664ca-81f2-429f-b288-97652b26ef13)

**Reply by spcqike** [reply]
sieht mir nach nem Anwenderfehler aus. die ANleitung sieht klar 3 schnelle 1A Sicherungen an den Spannungsmesseingängen vor.

ich habe einen 3EM in der Verteilung, auf Arbeit haben wir seit mehreren Jahren um die 30 Stück verbaut. noch ist nicht einer ausgestiegen. Weder eine der Sicherungen ist gekommen, geschweige denn der Shelly selber.

**Reply by Sorbat** [reply]
Ja, das kann natürlich auch sein. Aber dieses Problem soll wohl auftreten bei hohen Einschaltströmen zb durch Leuchtstoffröhren.

**Reply by spcqike** [reply]
Hm. Wir messen den Stromverbrauch diverser Industrieanlagen, hauptsächlich Spritzgussmaschienen. Die haben mal gut und gerne 30-40kW, große Heizungen, Motoren und hydraulikpumpen. Arbeitsplatzbeleuchtung hängt da genauso dran, wie normale Schuko Steckdosen für Handgeräte aber auch CEE Dosen für Granulattrockner. 

Dass eine Leuchtstoffröhre den Shelly da überfordert, kann ich mir nicht so recht vorstellen (: geschweige denn ein privat Haushalt allgemein. 

Nachtrag:
Zugegeben, wir sind mit unseren ~30 Geräten statistisch wahrscheinlich nicht sehr relevant. Aber sind das 10 in Foren gemeldete defekte Einheiten, bei wahrscheinlich Millionen verkauften?

**Reply by Sorbat** [reply]
Ja da hast du auch wieder recht. Aber ich baue den 3Em so oder so dieser Tage aus.

**Reply by Manos1966** [reply]
@Sorbat Hast du keinen digitalen Stromzähler?

**Reply by Sorbat** [reply]
> @Sorbat Hast du keinen digitalen Stromzähler?

Doch klar, ich versuche gerade mal den Pulse einzurichten 
Wieso? Was schwebt dir vor?

**Reply by Manos1966** [reply]
Sehr gut!
Dann die einfachste und genaueste Art die Daten zu lesen ist mit TASMOTA Lesekopf.
Ich habe zehn davon installiert bei Freunden und sie funktionieren einwandfrei.

Eine Suche nach HICHI Lesekopf in eBay wird dir helfen.
Preise liegen zwischen 10 und 40 EUR

Die beste Loesung meiner Meinung nach, ist ein Lesekopf mit WiFi integriert: Suche nach "TTL IR WIFI Lesekopf All-In-One Leseschreibkopf Volkszähler Smartmeter wie Hichi" circa 28EUR
- Einbindung  mit Stromzaehler ist etwas kompliziert aber gut beschrieben: Du muss das richtige Script fuer deine Stromzaehler finden https://tasmota.github.io/docs/Smart-Meter-Interface/#commands und in TASMOTA addieren (eine Beschreibung wird mit dem Lesekopf dabei sein)
- Einbindung mit OpenDTU ist Kinderleicht https://github.com/helgeerbe/OpenDTU-OnBattery/wiki/Power-Meter#tasmota

**Reply by Sorbat** [reply]
ich hatte eben den Pulse konfiguriert bekommen, das hat mit dem richtigen Build auf anhieb funktioniert. Aber das wird für mich auch nur eine Übergangslösung sein. 

Um die Hichi hatte ich bisher immer einen großen Bogen gemacht, weil mir das so bastellösungsmäßig vor kam. 
Weißt du denn zufällig in welchen Intervallen der Hichi Daten schickt?

**Reply by spcqike** [reply]
> Weißt du denn zufällig in welchen Intervallen der Hichi Daten schickt?

Kommt auf deine Konfiguration an. Von täglich bis „bei wert Änderung“ ist alles möglich. Wobei ich mir bei letzterem nicht sicher bin, ob das auch sekündlich oder gar öfter passiert. Hab es nicht getestet. 

> Um die Hichi hatte ich bisher immer einen großen Bogen gemacht, weil mir das so bastellösungsmäßig vor kam.

das ist genauso gebastelt wie ne openDTU. Von selbst gelötet, freiluft verkabelt bis fertig bestückte Leiterplatte in 3D Druck Gehäuse findet man alles. Software ist Open Source.
